.golangci.yml

```diff
@@ -0,0 +1,23 @@
+run:
+  skip-files:
+    - .*_test.go
+  timeout: 10m
+  go: '1.189
```

Comment:
[blocking][question] does this need to be configured? Since go `1.19` is specified in `go.mod` proposal to remove this line.

```
  # Define the Go version limit.
  # Mainly related to generics support since go1.18.
  # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.18
  go: '1.19'
```

Ref: https://golangci-lint.run/usage/configuration/#run-configuration

```suggestion
```